### PR TITLE
Remove retired wg labels

### PR DESCRIPTION
**Pull request by mrbobbytables**
ref: https://github.com/kubernetes/community/issues/5034
ref: https://github.com/kubernetes/community/issues/5254
ref: https://github.com/kubernetes/community/pull/4976

### CODE REVIEW

**Review by fejta — APPROVED** [review]
/hold

**Comment by nikhita** [comment]
@mrbobbytables looks like this is good to be merged? :)

**Comment by mrbobbytables** [comment]
@nikhita yup! thanks for the ping^^;;; I thought this was still missing lgtm/approve.
/hold cancel

**Pull request merged**
